### PR TITLE
GLT-2270: fix user and time in QC change log entries

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Box.java#L17](https://github.com/TGAC/miso-lims/blob/c8588921b050c07d71c12640a1193d19e633b922/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Box.java#L17): Move constants to a class or enum. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)
1. ![CRITICAL][CRITICAL] [ChangeLoggable.java#L14](https://github.com/TGAC/miso-lims/blob/c8588921b050c07d71c12640a1193d19e633b922/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ChangeLoggable.java#L14): Move constants to a class or enum. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)
1. ![CRITICAL][CRITICAL] [Run.java#L463](https://github.com/TGAC/miso-lims/blob/c8588921b050c07d71c12640a1193d19e633b922/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Run.java#L463): Null is returned but a "Boolean" is expected. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2447)
1. ![CRITICAL][CRITICAL] [DetailedSampleBuilder.java#L976](https://github.com/TGAC/miso-lims/blob/c8588921b050c07d71c12640a1193d19e633b922/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/DetailedSampleBuilder.java#L976): Define a constant instead of duplicating this literal "Cannot delete non-persisted builder object" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [PoolImpl.java#L567](https://github.com/TGAC/miso-lims/blob/c8588921b050c07d71c12640a1193d19e633b922/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L567): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: sqlstore/src/main/resources/db/migration/V8502__changeloggables.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SELECT userId INTO @admin FROM User WHERE loginName = 'admin';
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration/V8502__changeloggables.sql]: `--Start/EndNoTest`?

Comment (djcooke) [reply sqlstore/src/main/resources/db/migration/V8502__changeloggables.sql]: Yep. Also need to finish modifying test data scripts to include the new fields to fix some tests

**Comment by sqwci** [comment]
SonarQube analysis reported 6 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Box.java#L17](https://github.com/TGAC/miso-lims/blob/6235a30e562bd67f55c3c03ef5d5b1cee1cd0a9e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Box.java#L17): Move constants to a class or enum. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)
1. ![CRITICAL][CRITICAL] [ChangeLoggable.java#L14](https://github.com/TGAC/miso-lims/blob/6235a30e562bd67f55c3c03ef5d5b1cee1cd0a9e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ChangeLoggable.java#L14): Move constants to a class or enum. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)
1. ![CRITICAL][CRITICAL] [Run.java#L463](https://github.com/TGAC/miso-lims/blob/6235a30e562bd67f55c3c03ef5d5b1cee1cd0a9e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Run.java#L463): Null is returned but a "Boolean" is expected. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2447)
1. ![CRITICAL][CRITICAL] [DetailedSampleBuilder.java#L976](https://github.com/TGAC/miso-lims/blob/6235a30e562bd67f55c3c03ef5d5b1cee1cd0a9e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/DetailedSampleBuilder.java#L976): Define a constant instead of duplicating this literal "Cannot delete non-persisted builder object" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [PoolImpl.java#L567](https://github.com/TGAC/miso-lims/blob/6235a30e562bd67f55c3c03ef5d5b1cee1cd0a9e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L567): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [DefaultExperimentService.java#L209](https://github.com/TGAC/miso-lims/blob/6235a30e562bd67f55c3c03ef5d5b1cee1cd0a9e/miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultExperimentService.java#L209): Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sqwci** [comment]
SonarQube analysis reported 7 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Box.java#L17](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Box.java#L17): Move constants to a class or enum. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)
1. ![CRITICAL][CRITICAL] [ChangeLoggable.java#L14](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ChangeLoggable.java#L14): Move constants to a class or enum. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)
1. ![CRITICAL][CRITICAL] [Run.java#L463](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Run.java#L463): Null is returned but a "Boolean" is expected. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2447)
1. ![CRITICAL][CRITICAL] [DetailedSampleBuilder.java#L976](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/DetailedSampleBuilder.java#L976): Define a constant instead of duplicating this literal "Cannot delete non-persisted builder object" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![MAJOR][MAJOR] [DefaultStudyService.java#L139](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultStudyService.java#L139): Update this method so that its implementation is not identical to "getStudyStore" on line 64. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)
1. ![MINOR][MINOR] [PoolImpl.java#L567](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L567): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [DefaultExperimentService.java#L209](https://github.com/TGAC/miso-lims/blob/d889a3d1abbf0cc4bec8b35ec60933942691ed99/miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultExperimentService.java#L209): Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException' which is a runtime exception. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**